mediabridge/data_processing/wiki_to_netflix.py

```diff
@@ -95,6 +98,9 @@ def wiki_query(data_csv, user_agent):
     for row in tqdm(data_csv):
       if row[1] is None:
             continue
+          
+      if '"' in row[2]:
+            row[2] = row[2].replace('"','\\"')
```

Comment:
Just wanted to double check that you confirmed that `\"` is the proper way to escape quotes in SPARQL. I think I spitballed a bit and used that as an example, but I have no idea if it's correct.

Reply:
I went the route of replacing any double quotes with '\\"' in the titles that include doubles quotes (which escapes quotes) before these titles reach the SPARQL query portion. So instead of having the query deal with escaping the quotes, the titles are reformatted beforehand to escape this. I created a sample .txt file with titles that all have double quotes to test it out and it worked, but let me know if you're fine with me using this logic & going about it this way.

Reply:
Yes that's fine. Testing it directly is totally acceptable. Good work.